shared/src/sdk/signing.rs

```diff
@@ -364,14 +365,10 @@ pub async fn wrap_tx<
     };
     let fee_amount = match args.fee_amount {
         Some(amount) => {
-            let validated_fee_amount = validate_amount::<_, IO>(
-                client,
-                amount,
-                &args.fee_token,
-                args.force,
-            )
-            .await
-            .expect("Expected to be able to validate fee");
+            let validated_fee_amount =
```

Comment:
I just remember, are we allowed to have panics in the sdk still?

Reply:
Thanks. You're correct: panics are not allowed in the SDK. Since SDK integration is urgent - perhaps we can make a future PR where we identify the remaining occurrences of panics and fix them?